### DISCUSSION TITLE
插件区域，时间和日期中间有较大空白，有无考虑进行优化

### DISCUSSION THREAD

**Discussion by linuxlionking** [linuxdeepin/developer-center#3945] [General | 常规讨论]
![截图_选择区域_20230329125527](https://user-images.githubusercontent.com/7598566/228430622-3919e8e3-9726-472c-916d-6efd2048ec46.png)

如图示，当显示较多插件时，空白相当明显，不加以利用，岂不浪费

想问问官方对这块有啥规划，大家也可以谈谈想法

**Comment by BLumia** [comment]
这栏能自定义内容或者插件扩展的话可能会好一些吧。

cc @meiyixiang

**Comment by meiyixiang** [comment]
确实会有这个问题，但暂时没有考虑去优化这里，这里显示这种状态是由于UI和交互设计问题造成的，待V23release版本发布，和DDE桌面功能改造正式完成后可根据实际用户反馈对这里进行优化处理。

**Reply by linuxlionking** [reply]
了解, 可以，这块不紧急。正式发版后再优化即可

**Comment by meiyixiang** [comment]
这个问题我做了记录，暂时不会去考虑下需求优化这个dock需求

**Comment by meiyixiang** [comment]
如果谈我的想法的化，我认为可以给时间划定一个固定区域插件上下两行都可以驻留当一排显示的时候自动合并为一行，具体优化时需要详细去考虑。

**Reply by linuxlionking** [reply]
这种实现方式，起码听起来优于当前的方式

**Comment by meiyixiang** [comment]
嗯是的，这个更偏向与交互和页面布局的设置问题，等版本稳定后可以去考虑优化。
